apps/masterbots.ai/components/layout/sidebar/sidebar-link.tsx

```
@@ -1,256 +1,311 @@
-    </div>
-  )
+	if (isBrowse || isFilterMode) {
+		return (
+			<div className={cn('flex flex-col mb-2')}>
+				<div
+					className={cn(
+						'flex items-center p-2 cursor-pointer',
+						isActive && 'bg-gray-200 dark:bg-mirage',
+					)}
+					onClick={handleClickCategory}
+				>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)